Update destination-functions.md - console.log UI limit (#6760)
segmentio/segment-docs: src/connections/functions/destination-functions.md

##### Can I view console.log() outputs in Destination Functions?

Incorporating console.log() statements in your Destination Function code aids in debugging. However, logs generated by these statements will only be accessible in the [Event Delivery](/docs/connections/event-delivery/) view if the payloads encounter errors during processing. Logs from successfully processed payloads are not displayed.

##### What is the maximum data size that can be displayed in console.logs() when testing a Function?

> The test function interface has a 4KB console logging limit. Outputs surpassing this limit will not be visible in the user interface.

Reviewer: ```suggestion
The test function interface has a 4KB console logging limit. Outputs larger than this limit are not visible in the user interface.
```
